### DISCUSSION TITLE
Renaming default branch name from  master to main?

### DISCUSSION THREAD

**Discussion by Nechir-89** [Tribalnetwork/Portfolio-Network#259] [Q&A]
### GitHub is renaming default branch name from master to main, do you think we also need to rename it? why?

- [ ] We need to go with GitHub by changing master with main
- [ ] We are good and we don't need to change

**Share something or an idea...**

**Comment by ghost** [comment]
We should rename it to main, as it would allow us to keep up with GitHub's standards and reduce confusion.

**Reply by Nechir-89** [reply]
Please provide us with a resource to follow and rename the branch safely?

**Comment by ZhongweiL** [comment]
I think we should rename it to main, because running the command `git checkout main` is more intuitive when someone tries to access the default branch. I have tried to use the command to access the default branch only to realize that it was not the correct branch name. I found this doc for renaming the default branch. https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/changing-the-default-branch

**Comment by Nechir-89** [comment]
Thanks guys, soon we will update it to main and will let you know it.
